liferay-echo/liferay-portal
# LPS-205100 Hide Permissions option in sidebar for Web Content

## Motivation

[Link to ticket](https://liferay.atlassian.net/browse/LPS-205100)

### Changes

Hide the "Permissions" option from the sidebar for Web Content when the FF is active.

>[!NOTE]
>You should activate the FF: LPS-198959

## Test scenario 1

1. Go to Content & Data > Web Content.
2. Check if the Permission options appears in the sidebar.

## Results

![image](https://github.com/liferay-echo/liferay-portal/assets/93203423/8e49b7c6-d69f-4622-993d-2603e0712dbb)
